APIWorking.js

```diff
@@ -1,65 +1,80 @@
-                  let resultsType = res.Search[i].Type
-                  let resultsYear = res.Search[i].Year
-                  // code to create divs to hold search results
-                  const main = document.getElementById("main");
-                  const div = document.createElement("div")
-                  //Add search results to the text of the new div
-                  let textContent = document.createTextNode(`${resultsTitle} (${resultsType}, ${resultsYear})`);
-                  const newDiv = document.getElementById("main").appendChild(div);
-                  let poster = res.Search[i].Poster;
-                  let img = document.createElement("img");
-                  let linebr = document.createElement("br")
-                  img.src = poster;
-                  img.title = "movie poster (broken image link)";
-                  newDiv.appendChild(textContent)
-                  newDiv.appendChild(linebr)
-                  newDiv.appendChild(img)
-                  document.getElementById("main").appendChild(newDiv);
-                  newDiv.classList.add("results")
-                  
-                  // Console log test to make sure it's working:
-                  // console.log(`Title: ${resultsTitle} Type: ${resultsType} Year: ${resultsYear}`)
-                }
-              });
-            } else {
-                document.getElementById("main").innerText = "Please enter 3 or more characters!"
-            }};
-        }   
-          });
\ No newline at end of file
+  .getElementById("test")
+  .addEventListener("click", function fetchTest() {
+    //remove prior results if any
+    const node = document.getElementById("main");
+    node.querySelectorAll('*').forEach(n => n.remove());
+    //set up variables for search results, with url snippets
+    const title = "&s=" + document.getElementById("searchBox").value;
+    const radios = document.getElementsByName("type");
+    const year = `&y=${document.getElementById("yearDropdown").value}`
+    //make type equal to the checked radio button.
+    for (var radio of radios) {
+      if (!radio.checked) {
+        continue;
+      }
+
+      if (title.length < 5) {
+        document.getElementById("main").innerText = "Please enter 5 or more characters!"
+        return
+      }
+
+      const type = "&type=" + radio.value;
+      document.getElementById("main").innerText = ""
+      //run search with user inputs
+      fetch("http://www.omdbapi.com/?apikey=5d6248fb" + title + type + year)
+        //log results
+        .then(function (response) {
+          console.log(response);
+          //if response is NOT ok show response status
+          if (!response.ok) {
+            console.log(response.status);
+          }
+          return response.json();
+        })
+        .then(function (res) {
+          const resultLength = res.totalResults;
+          let resultsText = `Showing 10 of ${resultLength} results:` 
+
+          if (!resultLength) {
```

Comment:
`res.totalResults` was `undefined` if no results came back, so we can just check to see if this value is truthy and if yes, than display a different message. Additionally, if `totalResults` was `0` for some reason, this would also run and set the message that no results came back